### PR TITLE
[WIP] Rename attrs hash

### PR DESCRIPTION
**Pull request by dmgasper**

### CODE REVIEW

**Comment by ramereth** [comment]
@dmgasper can you please add our default ``Rakefile`` and also add some ChefSpec tests?

**Comment by ramereth** [comment]
[resource_from_hash (rename attrs_hash to new_resource.attrs_hash)](https://trello.com/c/6trunBM6/1290-resourcefromhash-rename-attrshash-to-newresourceattrshash)

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
+
+ChefSpec::Coverage.start! { add_filter 'resource_from_hash' }
+
+CENTOS_7 = {
+  platform: 'centos',
+  version: '7.4.1708',
```
Comment (ramereth) [inline spec/spec_helper.rb]: Let's go ahead set this to ``7`` please.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}.freeze
+
+CENTOS_6 = {
+  platform: 'centos',
+  version: '6.9',
+}.freeze
```
Comment (ramereth) [inline spec/spec_helper.rb]: Let's not worry about CentOS 6, please remove.

**Comment by dmgasper** [comment]
Sounds good to me

**Comment by dmgasper** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.1.0.  Have a nice day!

**Pull request merged**